int32 id
string name
float32 battery_level
geometry_msgs/Pose2D car_loc
geometry_msgs/Twist car_speed